pyproject.toml

```diff
@@ -27,7 +27,7 @@ capella-console-wizard = 'capella_console_client.cli.wizard:main'
 
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
+python = ">=3.8,<=3.12.2"
```

Comment:
Good start

Can you please also add 
- 3.12 to the metadata tags of project.toml a bit further up and
- 3.12 on the CI tests (https://github.com/capellaspace/console-client/blob/main/.github/workflows/ci.yml#L56)

On my phone now but I potentially also reference supported python versions in the /docs

thanks

Reply:
Shouldn't this be:
`python = ">=3.8,<=3.13"`
to make it more future-proof?

Python only introduces bugfixes and security updates within given minor version so it should be safe to do `<3.13`

Reply:
I have checked our stack (~500 Python packages, a lot of them geospatially oriented) and most of the packages do not set the upper bound to Python version with only some of them setting it to `<4.0`. Besides your package there is only one package in our stack that has the upper bound set to `<3.X` so it seems like a bad practice to set the upper bound this way.

For me personally it is way better when I can install the package on the newest Python version by default and if something breaks I can just report the issue.

Reply:
I like the `<4.0` suggestion + explicit classifiers from a maintenance perspective.

@Shaunakde - since I am currently already working in this repo I am going to go ahead and get this in (meaning this PR will be able to be closed).

Hope that works for you

Reply:
TY for link. I had some of these in the follow-on comments. Taking a look / rebasing so it contains all the changes and uploading here.

Reply:
Rebased changes from https://github.com/capellaspace/console-client/pull/69